platform-core-module/src/main/java/com/softicar/platform/core/module/user/password/UserPasswordGenerator.java

```diff
@@ -42,7 +44,7 @@ public String generatePassword() {
 		throw new SofticarUserException(CoreI18n.FAILED_TO_GENERATE_PASSWORD);
 	}
 
-	public void resetUserPassword(AGUser user) {
+	public void resetUserPassword(AGUser user, boolean showNewPassword) {
```

Comment:
where is the call for this method with parameter `false`?

Reply:
Its the user password reset action, currently admin get shown the new password.
To preserve this, I added this flag